back-end/legeno-around-here/src/main/java/wooteco/team/ittabi/legenoaroundhere/service/UserService.java

```diff
@@ -72,13 +74,23 @@ private Area findAreaBy(Long areaId) {
     public TokenResponse login(LoginRequest loginRequest) {
         User user = userRepository.findByEmail(new Email(loginRequest.getEmail()))
             .orElseThrow(() -> new NotExistsException("가입되지 않은 회원입니다."));
+        checkEmailAuth(user);
+        checkPassword(loginRequest, user);
+        return new TokenResponse(
+            jwtTokenGenerator.createToken(user.getEmailByString(), user.getRoles()));
```

Comment:
길면 두 줄로 나눠도 될 것 같아요~!

```suggestion
        String token = jwtTokenGenerator.createToken(user.getEmailByString(), user.getRoles())
        return new TokenResponse(token);
```

Reply:
반영했습니다! :)